CompilerSource/backend/GameData.cpp

```diff
@@ -592,5 +614,19 @@ GameData::GameData(deprecated::JavaStruct::EnigmaStruct *es,
 
 GameData::GameData(const buffers::Project &proj, const EventData* /*events*/):
     filename("") {
-  FlattenProto(proj, this);
+  
+  buffers::Project copy = proj;
```

Comment:
Yikes this looks bad. I don't really want us to copy the _entire_ game. Even if we do copy the entire game, are we sure this is the point in the build we want to do the copying? I'm actually not even sure if we already are copying the game before passing it to the RPC too. Josh had me make those parameters const for a reason.

Consider just removing the font after we are done. At the very least find how to make protobuf do a shallow copy only of the root tree node rather than the entire project.

Reply:
The only other place I can think of to add this is in every other reader and I didn't want to put it in 5 places. If you can find a better place feel free to move it.